src/main/java/com/swordglowsblue/artifice/api/builder/data/dimension/NoiseConfigBuilder.java

```diff
@@ -51,7 +51,9 @@ public NoiseConfigBuilder densityFactor(double densityFactor) {
         return this;
     }
 
-    @Deprecated
+    /**
+     * @deprecated use the double version instead
```

Comment:
You need both the annotation and the comment...

Reply:
Oups